ts/modules/marker-clusters.ts

```diff
@@ -0,0 +1,1948 @@
+//                             gridOffset.plotTop + currentY,
+//                             scaledGridSize,
+//                             scaledGridSize
+//                         )
+//                         .attr({
+//                             stroke: series.color,
+//                             opacity: 0.3,
+//                             'stroke-width': '1px'
+//                         })
+//                         .add()
+//                         .toFront();
+
+//                     text = chart.renderer
+//                         .text(
+//                             j + '-' + i,
+//                             gridOffset.plotLeft + currentX + 2,
+//                             gridOffset.plotTop + currentY + 7
+//                         )
+//                         .css({
+//                             fill: 'rgba(0, 0, 0, 0.7)',
+//                             fontSize: '7px'
+//                         })
+//                         .add()
+//                         .toFront();
+
+//                     series.debugGridLines.push(elem);
+//                     series.debugGridLines.push(text);
+//                 }
+//             }
+//         }
+//     }
+// }
+/* eslint-enable require-jsdoc */
+
+
+// Cluster symbol.
+SvgRenderer.prototype.symbols.cluster = function (
+    this: Highcharts.SVGRenderer,
+    x: number,
+    y: number,
+    width: number,
+    height: number
+): Highcharts.SVGElement {
+    var w = width / 2,
+        h = height / 2,
+        outerWidth = 1,
+        space = 1,
+        inner, outer1, outer2;
+
+    inner = this.arc(x + w, y + h, w - space * 4, h - space * 4, {
+        start: Math.PI * 0.5,
+        end: Math.PI * 2.5,
+        open: false
+    } as any);
+
+    outer1 = this.arc(x + w, y + h, w - space * 3, h - space * 3, {
+        start: Math.PI * 0.5,
+        end: Math.PI * 2.5,
+        innerR: w - outerWidth * 2,
+        open: false
+    } as any);
+
+    outer2 = this.arc(x + w, y + h, w - space, h - space, {
+        start: Math.PI * 0.5,
+        end: Math.PI * 2.5,
+        innerR: w,
+        open: false
+    } as any);
+
+    return outer2.concat(outer1, inner);
+};
+
+Scatter.prototype.getGridOffset = function (
+    this: Highcharts.Series
+): Highcharts.GridOffsetObject {
+    var series = this,
+        chart = series.chart,
+        xAxis = series.xAxis,
+        yAxis = series.yAxis,
+        plotLeft = 0,
+        plotTop = 0;
+
+    if (series.dataMinX && series.dataMaxX) {
+        plotLeft = xAxis.reversed ?
+            xAxis.toPixels(series.dataMaxX) : xAxis.toPixels(series.dataMinX);
+    } else {
+        plotLeft = chart.plotLeft;
+    }
+
+    if (series.dataMinY && series.dataMaxY) {
+        plotTop = yAxis.reversed ?
+            yAxis.toPixels(series.dataMinY) : yAxis.toPixels(series.dataMaxY);
+    } else {
+        plotTop = chart.plotTop;
+    }
+
+    return {
+        plotLeft: plotLeft,
```

Comment:
Unnecessary, we can now use shorthand `{ plotLeft, plotHeight }` that will be transpiled to ECMAScript 2015.